docs/azure-data-studio/quickstart-sql-database.md

```diff
@@ -36,7 +36,7 @@ Use Azure Data Studio to establish a connection to your Azure SQL Database serve
    
    ![Screenshot showing the Welcome Azure Delta Studio dialog box with the Next connection option called out.](media/quickstart-sql-database/new-connection-icon.png)
```

Comment:
Can you please change "Delta" to "Data" in the image description?  (I know that wasn't your original fix, but I noticed it while in here).  Thanks!